docs/canvas/canvas-function-reference.asciidoc

```diff
@@ -1,17 +1,2133 @@
+
+Filters rows in a `datatable` based on the return value of a subexpression.
+
+*Accepts:* `datatable`
+
+[cols="3*^<"]
+|===
+|Argument |Type |Description
+
+|_Unnamed_ *** 
+
+Aliases: `fn`, `exp`, `expression`
+|`boolean`
+|An expression to pass into each row in the `datatable.`
+The expression should return a `boolean`. A `true` value preserves the row, 
+and a `false` value removes it.
+|===
+
+*Returns:* `datatable`
+
+
+[float]
+[[filters_fn]]
+=== `filters`
+
+Collects element filters on the workpad, usually to provide them to a data source.
+
+*Accepts:* `null`
+
+[cols="3*^<"]
+|===
+|Argument |Type |Description
+
+|_Unnamed_ *†* 
+
+Alias: `group`
+|`string`
+|The name of the filter group to use
+
+|`ungrouped`
+
+Alias: `nogroup`, `nogroups`
+|`boolean`
+|Don't include filters that belong to groups
+
+Default: `false`
+|===
+
+*Returns:* `filter`
+
+
+[float]
+[[font_fn]]
+=== `font`
+
+Creates font style.
+
+*Accepts:* `null`
+
+[cols="3*^<"]
+|===
+|Argument |Type |Description
+
+|`size`
+|`number`
+|The font size in pixels 
+
+Default: `14`
+
+|`lHeight` 
+
+Alias: `lineHeight`
+|`number`, `null`
+|The line height in pixels
+
+|`family`
+|`string`
+|An acceptable CSS web font string 
+
+Default: `"'Open Sans', Helvetica, Arial, sans-serif"`
+
+|`color`
+|`string`
+|The text color
+
+|`weight`
+|`string`
+|The font weight, for example, `"normal"`, `"bold"`, `"bolder"`, `"lighter"`, 
+`"100"`, `"200"`, `"300"`, `"400"`, `"500"`, `"600"`, `"700"`, `"800"`, `"900"` 
+
+Default: `"normal"`
+
+|`underline`
+|`boolean`
+|Underline the text? 
+
+Default: `false`
+
+|`italic`
+|`boolean`
+|Italicize the text? 
+
+Default: `false`
+
+|`align`
+|`string`
+|The horizontal alignment of text 
+
+Default: `left`
+|===
+
+*Returns:* `style`
+
+
+[float]
+[[formatdate_fn]]
+=== `formatdate`
+
+Format a valid date string or number of milliseconds since epoch using https://momentjs.com/[MomentJS]
+
+*Accepts:* `number`, `string`
+
+[cols="3*^<"]
+|===
+|Argument |Type |Description
+
+|_Unnamed_ *** 
+
+Alias: `format`
+|`string`
+|MomentJS Format with which to bucket, see the https://momentjs.com/docs/#/displaying/[MomentJS documentation]
+|===
+
+*Returns:* `string`
+
+
+[float]
+[[formatnumber_fn]]
+=== `formatnumber`
+
+Turns a `number` into a `string` using a NumeralJS format.
+
+*Accepts:* `number`
+
+[cols="3*^<"]
+|===
+|Argument |Type |Description
+
+|_Unnamed_ *** 
+
+Alias: `format`
+|`string`
+|NumeralJS format string, see the http://numeraljs.com/#format[NumeralJS documentation]
+|===
+
+*Returns:* `string`
+
+
+[float]
+[[g_fns]]
+== G
+[[getCell_fn]]
+=== `getCell`
+
+Fetches a single cell in a table.
+
+*Accepts:* `datatable`
+
+[cols="3*^<"]
+|===
+|Argument |Type |Description
+
+|_Unnamed_ 
+
+Aliases: `column`, `c`
+|`string`
+|The name of the column value to fetch, defaults to the first column
+
+|`row` 
+
+Alias: `r`
+|`number`
+|The row number, starting at 0 
+
+Default: `0`
+|===
+
+*Returns:* Depends on the data in the cell
```

Comment:
Would it be helpful to specify which cell?

Reply:
how about `Depends on the type of the given column`?